content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -2,69 +2,166 @@
+----------------------
 
-After configuring which warehouse(s) to resupply from, a new route is now available on all product
-forms. The new route appears as :guilabel:`Supply Product from [Warehouse Name]` under the
-:guilabel:`Inventory` tab on a product form. Use the :guilabel:`Supply Product from [Warehouse
-Name]` route with a reordering rule or the make to order (MTO) route to replenish stock by moving
-the product from one warehouse to another.
+Products must also be configured properly in order for them to be transferred between warehouses.
 
-.. image:: resupply_warehouses/product-resupply-route-settings.png
-   :align: center
-   :alt: Route setting which enables a product to resupplied from a second warehouse.
+Go to :menuselection:`Inventory app --> Products --> Products` and select the desired product. In
+the :guilabel:`Inventory` tab, the new route appears as :guilabel:`X: Supply Product from Y` in the
+:guilabel:`Routes` section.
 
-When a product's reordering rule is triggered and the product has the :guilabel:`Supply Product
-from [Warehouse Name]` route set, Odoo automatically creates two pickings. One picking is a
-*delivery order* from the second warehouse, which contains all the necessary products, and the
-second picking is a *receipt* with the same products for the main warehouse. The product move from
-the second warehouse to the main warehouse is fully tracked in Odoo.
+.. note::
+   *X* is the store's warehouse that receives products, and *Y* is the warehouse that sends
+   products.
 
-On the picking/transfer records created by Odoo, the :guilabel:`Source Document` is the product's
-reordering rule. The location between the delivery order and the receipt is a transit location.
+Tick the :guilabel:`X: Supply Product from Y` checkbox, which is intended to be used with the |MTO|
+route or a reordering rule to replenish stock by moving the product from one warehouse to another.
+Proceed to the dedicated sections below to continue the process.
 
-.. image:: resupply_warehouses/resupply-receipts-from-reordering-rule.png
-   :align: center
-   :alt: A reordering rule automatically creates two receipts for stock between warehouses.
+MTO
+~~~
 
-.. image:: resupply_warehouses/second-warehouse-delivery-order.png
-   :align: center
-   :alt: A warehouse order for resupplying one warehouse's stock with another.
+To replenish products using the make to order method, go to the product form and ensure the
+:ref:`MTO route is unarchived <inventory/warehouses_storage/unarchive-mto>`, so it appears in the
+:guilabel:`Routes` section of the :guilabel:`Inventory` tab.
+
+With the resupply and |MTO| routes ticked, jump to the next :ref:`Resupply from another warehouse
+workflow <inventory/warehouses_storage/resupply-workflow>` section.
+
+.. example::
+   The product, sold at the warehouse, `Store`, is resupplied from the central warehouse, named
+   `YourCompany`. To replenish the product using |MTO|, the following routes are selected:
```

Comment:
Ditto above comment about |MTO| being weirdly formatted.

Reply:
This seemed ok for me.